examples/experimental/openvino/yolo_v5/main.py

```diff
@@ -0,0 +1,151 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import subprocess
+from pathlib import Path
+
+import openvino.runtime as ov
+import torch
+import nncf
+
+from yolov5.utils.general import check_dataset
+from yolov5.utils.general import download
+from yolov5.utils.dataloaders import create_dataloader
+from yolov5.val import run as validation_fn
+
+
+FILE = Path(__file__).resolve()
+# Relative path to the `yolo_v5` directory.
+ROOT = Path(os.path.relpath(FILE.parent, Path.cwd()))
```

Comment:
I'm pretty sure that everything can be done via `pathlib`, no need to involve `os.path`

Reply:
Done.